CONTRIBUTING.md

```diff
@@ -168,6 +177,32 @@ Please remember to:
 Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels ((enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/aws/aws-step-functions-data-science-sdk-python/labels/help%20wanted) issues is a great place to start.
 
 
+## Setting Up Your Development Environment
+
+### Setting Up Your Environment for Debugging
+
+Setting up your IDE for debugging tests locally will save you a lot of time.
+You might be able to `Run` and `Debug` the tests directly in your IDE with your default settings, but if it's not the case,
+follow the steps described in this section.
+
+#### PyCharm
+1. Set your Default test runner to `pytest` in _Preferences → Tools → Python Integrated Tools_
+1. If you are using `PyCharm Professional Edition`, go to _Preferences → Build, Execution, Deployment → Python Debugger_ and set the options with following values:
+   
+   | Option                                                      | Value                 |
+   |------------------------------------------------------------:|:----------------------|
```

Comment:
nit: is there a reason we are right-aligning the text? it looks odd in the rendered version imo

Reply:
I initially wanted to follow the same alignment as the other table, but it does look better left aligned since the text if longer - change included in latest commit :)